python-flask/app/extensions.py

```diff
@@ -0,0 +1,5 @@
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+
+db = SQLAlchemy()
+migrate = Migrate()
```

Comment:
Do we need migration since the DB is fixed in our case?

Reply:
^same

Reply:
Addressed and deleted Migrate. U're right guys!